.github/workflows/deploy_staging.yml

```diff
@@ -0,0 +1,48 @@
+name: Deploy staging app to Firebase App Distribution
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  deploy_android:
+    name: Deploy Android app to Firebase App Distribution
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    defaults:
+      run:
+        working-directory: ./sample
+    steps:
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+
+      - name: Checkout source code
+        uses: actions/checkout@v2.3.2
+
+      - name: Cache Gradle
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches/modules-*
+            ~/.gradle/caches/jars-*
+            ~/.gradle/caches/build-cache-*
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: Decode buildKonfig properties
+        run: echo ${{ secrets.BUILD_KONFIG_PROPERTIES }} | base64 --decode > buildKonfig.properties
```

Comment:
This config should be separated between [staging & production](https://github.com/nimblehq/kmm-templates/settings/environments). Please help to define 2 env config in this repo 🙏

Reply:
@luongvo As we use the same `BUILD_KONFIG_PROPERTIES`for both staging and production, do we need to separate them? 🤔